app/confirmation/get.controller.tests.js

```diff
@@ -14,118 +14,13 @@ const getApp = require('../../server').getApp
 const paymentFixtures = require('../../test/fixtures/payments-fixtures')
 const { CookieBuilder } = require('../../test/test_helpers/cookie-helper')
 
-describe('confirmation one-off payment get controller', function () {
+describe('confirmation get controller', function () {
   let response, $
   const mandateExternalId = 'sdfihsdufh2e'
-  const gatewayAccoutExternalId = '1234567890'
```

Comment:
lol to the spelling mistake. took me a few minutes to figure out what the diff is.